modules/getting-started-web-console-connecting-a-database.adoc

```diff
@@ -38,5 +36,16 @@ Once you mark the `national-parks-app` application as a backend for the map visu
 |`MONGODB_DATABASE`|`mongodb`
 |`MONGODB_ADMIN_PASSWORD`|`mongodb`
 |===
++
+[TIP]
+====
+Click *Add value* to add each additional environment variable.
+====
+
+. In the *Advanced options* section, deselect *Create a route*.
```

Comment:
```suggestion
. In the *Advanced options* section, clear *Create a route*.
```
(if appropriate) - If this is a checkbox, per IBM, "Use clear to refer to deselecting the choice."